config.toml

```diff
@@ -68,6 +68,7 @@ description = "The CNCF Cloud Native Glossary Project is intended to be used as
 #time_format_default = "02.01.2006"
 #time_format_blog = "02.01.2006"
 
+
```

Comment:
We don't have to modify this part, but I guess it was from previous manual rebase.

In order to minimize mixed up situation, we'd better **update dev-de first**, and then, let's merge this PR.